client/src/components/CommentsPage.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import axios from 'axios';
+import { Router, Route, Link, IndexRoute, hashHistory, browserHistory, DefaultRoute, IndexLink } from 'react-router';
+
+import CommentsBanner from './CommentsBanner.jsx';
+import CommentsChat from './CommentsChat.jsx';
+
+class CommentsPage extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      university: {}
+    };
+  }
+
+  componentDidMount() {
+    this.getBanner(1);
```

Comment:
1 is a placeholder, will add the react router route param once we have that implemented.